.github/workflows/build_rpms.yml

```diff
@@ -0,0 +1,50 @@
+name: RPM Build
+
+on: [pull_request]
+
+jobs:
+  rpm_build:
+    name: "Build RPMs"
+    runs-on: ubuntu-latest
+    container: registry.gitlab.com/majorhayden/containers/rpm-builder:latest
```

Comment:
Is there a reason to use private container instead of the official fedora one? I can see you install dnf/git/rpmbuild/etc. below...